core/src/main/java/io/substrait/type/YamlRead.java

```diff
@@ -25,7 +26,8 @@ public class YamlRead {
 
   public static void main(String[] args) throws Exception {
     try {
-      System.out.println("Read: " + YamlRead.class.getResource("/functions_boolean.yaml"));
+      System.out.println(
```

Comment:
nit: @vbarua just curious do we need a println here? Or should we have used logs?

Reply:
This looks to be some old debugging code. Trying to run the main function throws a `FileNotFoundException` for me, so I'm not sure if it works anymore. I could be worth removing, but I think that would be better as it's own change.